src/server/replica.cc

```diff
@@ -1112,21 +1115,16 @@ error_code Replica::ParseAndExecute(base::IoBuf* io_buf) {
   uint32_t consumed = 0;
   RedisParser::Result result = RedisParser::OK;
 
-  io::NullSink null_sink;  // we never reply back on the commands.
-  ConnectionContext conn_context{&null_sink, nullptr};
-  conn_context.is_replicating = true;
-
   do {
     result = parser_->Parse(io_buf->InputBuffer(), &consumed, &resp_args_);
 
     switch (result) {
       case RedisParser::OK:
         if (!resp_args_.empty()) {
-          VLOG(2) << "Got command " << ToSV(resp_args_[0].GetBuf()) << ToSV(resp_args_[1].GetBuf())
-                  << "\n consumed: " << consumed;
+          VLOG(2) << "Got command " << ToSV(resp_args_[0].GetBuf()) << "\n consumed: " << consumed;
```

Comment:
I needed to remove the second arg print cause it causes crash when there is no second arg